src/en/news/blog/2022/s3select-intro/index.md

```diff
@@ -0,0 +1,74 @@
+---
+title: "s3 select - introduction"
+date: "2022-11-02"
+author: "gsalomon"
+---
+
+![s3select illustration](images/s3select-into.png)
+
+## What is ceph/s3select?
+On July 2020, the s3select was introduced to [CEPH upstream](https://github.com/ceph/s3select), an open-source package.
+
+The s3select is another S3 request, that enables the client to push down an SQL statement(according to AWS spec) into CEPH storage.
+
+## Why do we need that? What is it good for?
+To answer that, It needs to elaborate a bit on the push-down paradigm.
+
+The push-down paradigm is about moving(“pushing”) the operation close to the data.
+
+It's contrary to what is commonly done, i.e. moving the data to the “place” of operation.
+
+In a big-data ecosystem, it makes a big difference. 
+
+In order to execute 
+```SQL
+select sum( x + y) from s3object where a + b > c
+```
+
+It needs to fetch the entire object to the client side, and only then execute the operation with some analytic application,
+
+With push-down(s3-select) the entire operation is executed on the server side and only the result is returned to the client side.
+
+It should be noted that pushing down operations, close to data is not an easy “thing” to do, the storage is usually fragmented, and broken into random pieces.
+
+It makes it difficult to execute single-query on top of many random pieces, moreover, the type of these objects could be binary, in some cases.
+
+## What is s3select capable of?
+CEPH S3 select is an SQL-like engine, its properties can be reviewed [here](https://docs.ceph.com/en/latest/radosgw/s3select/#).
+
+The s3select features do not turn the s3 storage into a database, but they improve greatly the efficiency of SQL processing.
+
+The s3select property is embedded into the GetObject module which makes it highly efficient for the push-down operation.
+
+Upon an SQL query being push-down, the object is fetched (the same as with the GetObject request), and each fragment of the object is processed by the s3select module.
+
+Since the s3select is embedded into the S3 system, there are no redundant copies of bytes, the object is processed immediately, and the results send back to the client.
+
+Moreover, the s3select system is capable to process the following object types CSV, JSON, and Parquet.
+
+The same engine is processing these different object types.
```

Comment:
should we mention the limitation on processing only a single object (no JOIN operators)?  maybe you do later...

Reply:
in cephalocon talk, I do mention the limitation of single file processing,
and why it is wrong.
I also mentioned that our plan is not to limit that.

to add that section? what do you think?

as for JOIN operator, yes, we should discuss how to present that (JOIN execution), 
since it requires in some use cases extensive processing (it loads the storage nodes)
much more than the current load.